content/hi/docs/concepts/policy/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "नीतियाँ"
+weight: 90
+no_list: true
+description: >
+  नीतियों के साथ सुरक्षा और सर्वोत्तम प्रथाओं को प्रबंधित करें।
+---
+
+<!-- overview -->
+
+Kubernetes नीतियाँ वे कॉन्फ़िगरेशन होती हैं जो अन्य कॉन्फ़िगरेशन या रनटाइम व्यवहारों को प्रबंधित करती हैं। Kubernetes विभिन्न प्रकार की नीतियाँ प्रदान करता है, जो नीचे दी गई हैं:
+
+<!-- body -->
+
+## API ऑब्जेक्ट्स का उपयोग करके नीतियाँ लागू करें
+
+कुछ API ऑब्जेक्ट्स नीतियों के रूप में कार्य करते हैं। यहाँ कुछ उदाहरण दिए गए हैं:
+* [NetworkPolicies](/docs/concepts/services-networking/network-policies/) का उपयोग किसी वर्कलोड के लिए इनग्रेस और एग्रेस ट्रैफिक को प्रतिबंधित करने के लिए किया जा सकता है।
+* [LimitRanges](/docs/concepts/policy/limit-range/) विभिन्न ऑब्जेक्ट प्रकारों के बीच संसाधन आवंटन सीमाओं का प्रबंधन करते हैं।
+* [ResourceQuotas](/docs/concepts/policy/resource-quotas/) किसी {{< glossary_tooltip text="नेमस्पेस" term_id="namespace" >}} के लिए संसाधन खपत को सीमित करती हैं।
+
+## Admission Controllers का उपयोग करके नीतियाँ लागू करें
+
+एक {{< glossary_tooltip text="admission controller" term_id="admission-controller" >}} API सर्वर में चलता है और API अनुरोधों को सत्यापित या बदल सकता है। कुछ admission controllers नीतियों को लागू करने के लिए कार्य करते हैं। उदाहरण के लिए, [AlwaysPullImages](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) admission controller एक नए Pod में इमेज पुल नीति को `Always` पर सेट करने के लिए संशोधित करता है।
```

Comment:
```suggestion
एक {{< glossary_tooltip text="admission controller" term_id="admission-controller" >}} API सर्वर में चलता है और API अनुरोधों को सत्यापित या बदल सकता है। कुछ admission controllers नीतियों को लागू करने के लिए कार्य करते हैं। उदाहरण के लिए,[AlwaysPullImages](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) admission controller इमेज पुल नीति को `Always` पर सेट करने के लिए एक नए पॉड को संशोधित करता है।
```

Reason for the changes - 
- The modified changes specify that new pod That is scheduled on some node must have Image pull policy to set AlwaysPullImages regardless of the pre-defined image pull policy in PodSpec, if in cluster AlwaysPullImages admission controller is enabled.

Reply:
Reason for the changes -

The modified changes specify that new pod That is scheduled on some node must have Image pull policy to set AlwaysPullImages regardless of the pre-defined image pull policy in PodSpec, if in cluster AlwaysPullImages admission controller is enabled.

Reply:
Fix the issue